docs/farming-&-staking/farming/advanced-cli/cli-prerequisites.mdx

```diff
@@ -80,3 +80,15 @@ You can use any file system with default settings, we advise against their use f
 In order to disable **CoW** on **BTRFS**, run the command `chattr +C /your_farmer_directory/`.
 
 In case of **ZFS**, you can try adjusting the **recordsize** by running `zfs set recordsize=128K tank/dataset`, or creating a separate **dataset** for the farmer with adjusted **recordsize** parameters.
+
+
+### Folder structure
```

Comment:
How important is this information to users? They can see the structure when they run software anyway.

Reply:
It's important in the context of keypair location for operators, but I agree that they look a bit out-of-context here. Removed.